sql/2020/07_SEO/pages_wpt_bodies_hreflang_by_device_and_http_header_value.sql

```diff
@@ -11,15 +11,17 @@ CREATE TEMPORARY FUNCTION get_wpt_bodies_info(wpt_bodies_string STRING)
 RETURNS STRUCT<
     hreflangs ARRAY<STRING>
 > LANGUAGE js AS '''
-var result = {};
+var result = {
+  hreflangs: []
+};
 
 try {
     var wpt_bodies = JSON.parse(wpt_bodies_string);
 
     if (Array.isArray(wpt_bodies) || typeof wpt_bodies != 'object') return result;
 
     if (wpt_bodies.hreflangs && wpt_bodies.hreflangs.http_header && wpt_bodies.hreflangs.http_header.values) {
-        result.hreflangs = wpt_bodies.hreflangs.http_header.values;
+        result.hreflangs = wpt_bodies.hreflangs.http_header.values.map(v=> v); // seems to fix a coercion issue!
```

Comment:
🤨

Reply:
Strange, it's as if the array from the map was better than the original array. I tested that values was always an array of string?

Reply:
Strange indeed, but whatever works